pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -345,6 +347,28 @@ func (p *AnacondaInstallerISOTree) serialize() osbuild.Pipeline {
 		}
 
 		pipeline.AddStage(osbuild.NewKickstartStage(kickstartOptions))
+
+		if p.WheelNoPasswd {
+			// Because osbuild core only supports a subset of options,
+			// we append to the base here with hardcoded wheel group with NOPASSWD option
+			hardcodedKickstartBits := `
+%include /run/install/repo/osbuild-base.ks
+
+%post
+echo -e "%wheel\tALL=(ALL)\tNOPASSWD: ALL" > "/etc/sudoers.d/wheel"
+chmod 0440 /etc/sudoers.d/wheel
+restorecon -rvF /etc/sudoers.d
+%end
+`
+			kickstartFile, err := fsnode.NewFile(p.KSPath, nil, nil, nil, []byte(hardcodedKickstartBits))
+			if err != nil {
+				panic(err)
+			}
```

Comment:
When this is true, the kickstart options above (the base kickstart) should also change so that the path is `/osbuild-base.ks`.  The way it's written now in the PR, both the hardcoded kickstart here and the original kickstart will be written to `p.KSPath` (`/osbuild.ks`) and the second one will overwrite the primary.

Because we can't add an `%include` in the base kickstart currently, we need to write the primary one as `osbuild-base.ks` and add the include here, so this is correct, but Line 337 should change to `"osbuild-base.ks"`.

Reply:
done